tools/extractReleaseNotes.js

```diff
@@ -0,0 +1,48 @@
+// Parse PR descriptions for release notes
+const fetch = require("node-fetch");
+const fs = require("fs");
+
+// Set this path to the cached json file to avoid hitting the github api limit while developing
+const CACHED_RESPONSE_PATH = "";
+// Adjust this variable to the last PR that shouldn't be included in the release notes
+const LAST_PR = 0;
+
+const RELEASE_NOTES_REGEX = /### Mailable description[^:]*:\r\n([^#]*)/;
```

Comment:
Does the api always return `\r\n` for line breaks? How about `\r?\n` ?
What happens if an issue is referenced in the mailable description?

Won't the regex stop at the # ? Or is this not supposed to happen anyway?